src/contracts/relay-adapt/__tests__/relay-adapt.test.ts

```diff
@@ -1154,9 +1154,7 @@ describe('relay-adapt', function test() {
     populatedTransactionGasEstimate.from = DEAD_ADDRESS;
     await expect(
       RelayAdaptV2Contract.estimateGasWithErrorHandler(provider, populatedTransactionGasEstimate),
-    ).to.be.rejectedWith(
-      `RelayAdapt multicall failed at index 0 with 'execution reverted (unknown custom error)': Unknown Relay Adapt error.`,
-    );
+    ).to.be.rejectedWith('RelayAdapt multicall failed at index 0.');
```

Comment:
The error message was updated in the implementation, so updating it in tests too.